tests/test_rubicon.py

```diff
@@ -1160,3 +1160,21 @@ def test_update3(self):
         self.assertEqual(d['four'], 'FIVE')
         self.assertEqual(len(d), len(self.py_dict) + 1)
 
+
+class BlockTests(unittest.TestCase):
+    def test_block_property(self):
+        BlockPropertyExample = ObjCClass("BlockPropertyExample")
+        instance = BlockPropertyExample.alloc().init()
+        result = ObjCBlock(instance.blockProperty, c_int, c_int, c_int)(1, 2)
```

Comment:
This part of the API gives me pause; it's the only place in public API that we'd be publicly referencing ctypes raw types. Could/should this refer to Python types that we then convert to their native variants?